frontend/src/components/Login.js

```diff
@@ -1,10 +1,110 @@
 import React from 'react';
+import {useState} from 'react'
+import axios from 'axios';
+import jwt_decode from "jwt-decode";
+import {TextField, Button, Typography, Grid, DialogContentText, Link} from '@mui/material';
+import SideBar from "./SideBar";
+import TopBar from "./TopBar";
+import StickyFooter from "./Footer";
 
-const Login = () => {
-    return (
-        <div>
-        </div>
-    )
+const Login = ({userInformations, setUserInformations}) => {
+    const [errorMessage, setErrorMessage] = useState()
+
+    const logUserIn = (evt) => {
+        // Prevents the form from being submitted
+        evt.nativeEvent.preventDefault();
+
+        // Get the form fields
+        const formElements = evt.nativeEvent.target.elements;
+        const email = formElements.email.value;
+        const password = formElements.password.value;
+
+        axios ({
+            method: 'POST',
+            baseURL: 'http://localhost:8080',
```

Comment:
Hardcoded host URL, is there a way to use a environment variable instead ?

Reply:
Added a TODO, will fix it later as for right now the backend is configured to launch on the given base URL.